src/components/Quiz/QuizzesStats.tsx

```diff
@@ -0,0 +1,184 @@
+import { trackCustomEvent } from "../../utils/matomo"
+
+import { QuizShareStats } from "../../types"
+
+import { TOTAL_QUIZZES_NUMBER, TOTAL_QUIZZES_POINTS } from "../../constants"
+
+const handleShare = ({ score, total }: QuizShareStats) => {
+  shareOnTwitter({
+    score,
+    total,
+  })
+
+  trackCustomEvent({
+    eventCategory: "quiz_hub_events",
+    eventAction: "Secondary button clicks",
+    eventName: "Twitter_share_stats",
+  })
+}
+
+const QuizzesStats: React.FC = () => {
+  const { score: userScore, completed, average } = useContext(QuizzesHubContext)
+  const numberOfCompletedQuizzes = getNumberOfCompletedQuizzes(
+    JSON.parse(completed)
+  )
+
+  const computedAverage =
+    average.length > 0 ? average.reduce((a, b) => a + b, 0) / average.length : 0
+  const parsedAverage = Number.isInteger(computedAverage)
+    ? computedAverage
+    : computedAverage.toFixed(2)
+
+  return (
+    <Box flex={1} order={{ base: 1, lg: 2 }} w="full">
+      <Stack mt={{ base: 0, lg: 12 }} gap={{ base: 8, lg: 4 }}>
+        {/* user stats */}
+        <Grid
+          gap={4}
+          bg="backgroundHighlight"
+          borderRadius={{ base: "none", lg: "lg" }}
+          border="none"
+          p={4}
+          mb={-2}
+        >
+          <GridItem colSpan={{ base: 2, lg: 1 }} alignSelf="center" order={1}>
+            <Text
+              color="body"
+              fontWeight="bold"
+              fontSize="xl"
+              margin={0}
+              textAlign={{ base: "center", lg: "left" }}
+            >
+              <Translation id="your-total" />
+            </Text>
+          </GridItem>
+
+          <GridItem
+            colSpan={{ base: 2, lg: 1 }}
+            justifySelf={{ base: "auto", lg: "end" }}
+            alignSelf="center"
+            order={{ base: 3, lg: 2 }}
+          >
+            <Button
+              variant="outline-color"
+              leftIcon={<FaTwitter />}
+              onClick={() =>
+                handleShare({ score: userScore, total: TOTAL_QUIZZES_POINTS })
+              }
+              w={{ base: "full", lg: "auto" }}
+              mt={{ base: 2, lg: 0 }}
+            >
+              <Translation id="share-results" />
+            </Button>
+          </GridItem>
+
+          <GridItem colSpan={2} order={{ base: 2, lg: 3 }}>
+            <Stack gap={2}>
+              <Flex
+                justifyContent={{ base: "center", lg: "flex-start" }}
+                alignItems="center"
+              >
+                <Circle size="64px" bg="primary" mr={4}>
+                  <TrophyIcon color="neutral" w="35.62px" h="35.62px" />
+                </Circle>
+
+                <Text fontWeight="bold" fontSize="5xl" mb={0} color="body">
+                  {userScore}
+                  <Text as="span" color="bodyMedium">
+                    /{TOTAL_QUIZZES_POINTS}
+                  </Text>
+                </Text>
+              </Flex>
+
+              <Progress value={userScore} />
```

Comment:
```suggestion
              <Progress value={(userScore / TOTAL_QUIZZES_POINTS) * 100} />
```

<img width="1042" alt="Screenshot 2023-06-02 at 16 38 29" src="https://github.com/ethereum/ethereum-org-website/assets/62268199/8dce6a02-b7f0-4de0-a513-a2c1cc2427f5">

Reply:
Thanks, fixed